src/components/Elements.test.tsx

```diff
@@ -5,11 +5,11 @@ import {renderHook} from '@testing-library/react-hooks';
 import {
   Elements,
   useElements,
-  useStripe,
   ElementsConsumer,
   useCartElement,
   useCartElementState,
 } from './Elements';
+import {useStripe} from './CustomCheckout';
```

Comment:
Would prefer to keep useStripe where it was

Reply:
Fixed here https://github.com/stripe/react-stripe-js/pull/439/commits/d16662ad985721528520d1cb959319b1cfe00fcd